src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedQueryBuilder.cs

```diff
@@ -192,7 +188,7 @@ public string BuildGetPackageUri(PackageIdentity package)
             // Start - Chocolatey Specific Modification
             //////////////////////////////////////////////////////////
 
-            var filterParameter = BuildFilter(searchTerm, filters.Filter, filters.ExactPackageId);
+            var filterParameter = BuildFilter(searchTerm, filters, true);
```

Comment:
Just to make it a bit clearer
```suggestion
            var filterParameter = BuildFilter(searchTerm, filters, includeSearchClauses: true);
```